shared/actions/chat/creators.js

```diff
@@ -7,7 +7,7 @@ import HiddenString from '../../util/hidden-string'
 import {List, Map} from 'immutable'
 import {chatTab} from '../../constants/tabs'
 import {setRouteState} from '../route-tree'
-import {uniq} from 'lodash'
+import uniq from 'lodash/uniq'
```

Comment:
mmm, i debated it and just went with doing it manually. kinda feel mixed about it. could go either way